misc/analysis/roc_curve.py

```diff
@@ -0,0 +1,371 @@
+import seaborn as sns
+sns.set(style="darkgrid")
+
+import numpy as np
+from sklearn import metrics
+import matplotlib.pyplot as plt
+from sklearn.metrics.ranking import _binary_clf_curve
+import sys
+import pandas
+from matplotlib import rc
+
+if len(sys.argv) < 2:
+    print("please specify csv file as the first argument")
+
+analysis_file = sys.argv[1]
+
+# all parameters
+
+IgA_name = 'IgA_ratio_of_q0.5_of_means'
+IgM_name = "IgM_ratio_of_q0.5_of_means"
+IgG_name = "IgG_ratio_of_q0.5_of_means"
+score_names = [IgA_name, IgG_name, IgM_name]
+label_name = "cohort_type"
+plate_name = "plate_name"
+time_name = "days_after_onset"
+ctype_name = "cohort_id"
+
+time_thresholds = ((0, 10), (11, 14), (15, 10000))
+time_thresholds_roc = ((15, 10000), )
+
+cr2marker = {1: "<",
+             2: "*",
+             10: "*"}
+
+remove_IgA_for_quality_control = ("B92", "B60", "B22", "B103", "CMV11", "CMV30", "EBV32", "EBV51", "EBV54", "EBV59")
```

Comment:
Quick question because this relates to what we write in the manuscript:
Why are the CMV and EBV samples excluded? Is this also due to the dotty staining pattern or due to something else?

Reply:
Yes, this was vibors manual assessment.

Reply:
Ok, so it's more than 4.